### PR TITLE
feat(client): support `swcli job` command

### PR DESCRIPTION
**Pull request by tianweidut**
## Description
- depend on: https://github.com/star-whale/starwhale/pull/2071
- changes:
  - refactor e2e for github action
  - support `swcli job` sub commands
  - refactor pfp、speech examples with @predict and @evaluate decorator  
- command example:
  ```console
  > swcli job --help
  Usage: swcli job [OPTIONS] COMMAND [ARGS]...
  Job management, list/info/cancel/resume job
  Options:
    --help  Show this message and exit.

  Commands:
    cancel       Cancel job
    info         Inspect job details
    list (ls)    List all jobs in the specified project
    pause        Pause job
    recover      Recover removed job
    remove (rm)  Remove job
    resume       Resume job
  ``` 

## Modules
- [x] Client

## Checklist
- [x] run code format and lint check
- [x] add unit test
- [x] add necessary doc

### CODE REVIEW

File: client/starwhale/core/model/view.py [hunk 1]
```diff
@@ -113,6 +120,37 @@ def history(self, fullname: bool = False) -> t.List[t.Dict[str, t.Any]]:
...
+            model_uri=model_uri,
+            dataset_uris=dataset_uris,
+            runtime_uri=runtime_uri,
+            resource_pool=resource_pool,
+            run_handler=run_handler,
+        )
+        if ok:
+            console.print(":clap: success to create job")
+            console.print(
+                f":bird: visit web: {project_uri.full_uri}/evaluation/{version_or_reason}"
```
Comment (goldenxinxing) [inline client/starwhale/core/model/view.py:144]: The current eval job details is: http://{ip}/projects/{projectId}/evaluations/{jobId}

**Comment by goldenxinxing** [comment]
others lgtm

File: client/starwhale/core/model/view.py (hunk 1 above)
Comment (tianweidut) [reply client/starwhale/core/model/view.py]: fixed

Comment (tianweidut) [reply client/starwhale/core/model/view.py]: 👏 select server instance
👏 success to create job
🐦 visit web: http://127.0.0.1:8082/projects/starwhale/evaluations/1
🐒 run command: swcli job info http://127.0.0.1:8082/project/starwhale/job/1

**Review by goldenxinxing — APPROVED** [review]
LGTM

**Pull request merged**